src/hooks/usePressScaleAnimation.ts

```diff
@@ -0,0 +1,27 @@
+import { useCallback, useRef } from 'react'
+
+import { Animated } from 'react-native'
+
+export const usePressScaleAnimation = (scaleTo = 0.97) => {
+  const scale = useRef(new Animated.Value(1)).current
+
+  const startPress = useCallback(() => {
+    Animated.timing(scale, {
+      toValue: scaleTo,
+      duration: 100,
```

Comment:
Can we switch this to spring physics or use easing https://reactnative.dev/docs/easing#ease

Reply:
and for web consistency, 
https://github.com/AudiusProject/stems/blob/master/src/components/Button/Button.module.css#L8
https://github.com/AudiusProject/stems/blob/master/src/assets/styles/animations.css#L9

Reply:
It's already using easing in-out by default, but if you want I can make it explicit?
https://reactnative.dev/docs/animated#configuring-animations
> Animated.timing() animates a value over time using easing functions.
> By default, it uses a symmetric easeInOut curve that conveys the gradual acceleration of an object to full speed and concludes by gradually decelerating to a stop.

Reply:
nice nvm